tests/ffi/selinux/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+. ../common/prepare.sh
+
+disk_cleanup
+prepare_test
+reload_config
+
+#Preset with expected value the variable.
+expected_value="setenforce:  security_setenforce() failed:  Permission denied"
+
+# Get setenforce 0 permission denied from qm.
+setenforce_0=$(podman exec -it qm setenforce 0)
```

Comment:
I noticed this comment https://github.com/containers/qm/issues/368#issuecomment-2063909398
@dougsland @Yarboa Do we need to add setenforce 0 into [ffi-tools](https://gitlab.com/CentOS/automotive/container-images/ffi-tools/) ?

Reply:
My code is really basic checkup of the qm, its not even a tool.
Its not even FFI, its more like qm sanity.

Reply:
no, @nsednev is correct